components/Login/LoginPageMain.tsx

```diff
@@ -79,8 +79,9 @@ const LoginPageMain: React.FC<LoginPageMainProps> = ({
             <label className={styles.chkboxLabel}>Remember Me</label>
           </form>
           <div>
-            {/* TODO: Add forgot password functionality */}
-            <a className={styles.forgotPassword}> </a>
+            <a className={styles.forgotPassword} href="http://localhost:3000/forgotpassword">
```

Comment:
since we are going to deploy this, we can't just use an href to localhost. At some point in the component, do something like `const url = process.env.REACT_APP_BASE_URL|| 'http://localhost:8080'`. Then, on line 82, we can do something like `href = {url + "/forgotpassword"}`